clearwater_config_manager/sas_json_plugin.py

```diff
@@ -28,7 +34,26 @@ def file(self):
         return _file
 
     def default_value(self):
-        return _default_value
+        # If a value is set for sas_server, we should resolve it and set that as the default
+        ip_addrs = [subprocess.check_output(['bash', '-c', '. /etc/clearwater/config; echo -n $sas_server'])]
+        sas_use_signaling_interface = subprocess.check_output(
+            ['bash', '-c', '. /etc/clearwater/config; echo -n $sas_use_signaling_interface'])
+
+        if ip_addrs[0]: # pragma: no cover
+            try:
+                # Check if it's an IP address
+                ipaddress.ip_address(ip_addrs[0])
+            except ValueError:
+                # It's not an IP address, so try and resolve as a domain address.
+                ip_addrs = self.resolve_domain_address_in_namespace(ip_addrs[0], sas_use_signaling_interface)
+
+            sas_config = {"sas_servers": [{"ip": addr} for addr in ip_addrs]}
+            sas_json = json.dumps(sas_config)
+        else:
+            # No value set for sas_server, so returning an empty config file.
+            sas_json = _default_value
```

Comment:
It might be cleaner to always do `json.dumps` and get rid of the `_default_value`.